### PR TITLE
fs fuse kv driver refactor"

### PR DESCRIPTION
**Pull request by ElsieFan**

### CODE REVIEW

File: pkg/fs/client/kv/kv_interface.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package kv
+
+import "time"
+
+type KvCache interface {
```
Comment (luoyuedong) [inline pkg/fs/client/kv/kv_interface.go]: KvCache改成Client吧

File: pkg/fs/client/kv/kv_interface.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	Name() string
+	Get(key []byte) ([]byte, bool)
+	Set(key, value []byte) error
+	Dels(keys ...[]byte) error
+	ScanValues(prefix []byte) (map[string][]byte, error)
+}
+
+type MetaConfig struct {
```
Comment (luoyuedong) [inline pkg/fs/client/kv/kv_interface.go]: 叫Config吧

File: pkg/fs/client/vfs/vfs.go [hunk 3]
```diff
@@ -94,7 +95,7 @@ func WithOwner(uid, gid uint32) Option {
 	}
 }
 
-func WithMetaConfig(m meta.Config) Option {
+func WithMetaConfig(m kv.MetaConfig) Option {
```
Comment (luoyuedong) [inline pkg/fs/client/vfs/vfs.go]: WithKvConfig?

Comment (ElsieFan) [reply pkg/fs/client/vfs/vfs.go]: remain unchanged

**Review by luoyuedong — APPROVED** [review]

**Pull request merged**